rocksdb_admin/admin_handler.cpp

```diff
@@ -94,18 +94,21 @@ const char kKafkaConsumerType[] = "rocksplicator_message_consumer";
 const char kKafkaWatcherName[] = "RocksplicatorMessageConsumer";
 const uint32_t kKafkaConsumerPoolSize = 1;
 
+// Global map to keep track of serverset file watchers
+  std::unordered_map<std::string, std::shared_ptr<
+      ::kafka::KafkaBrokerFileWatcher>> serverset_path_to_file_watcher_map;
+
 // Global condition variables for kafka synchronization.
 struct SyncVariables
 {
   SyncVariables(): flag(false) {} // init flag to false
   std::atomic<bool> flag;
```

Comment:
we have the mutex to serialize the operation so the std::atomic is not needed?

Reply:
Done.